### ISSUE TITLE
Review accessibility properties used in default template

### ISSUE THREAD

**Issue by Mephisztoe** [dotnet/maui#5487]
### Description

When creating a .NET Maui application from scratch using the provided project template, the MainPage contains accessibility properties such as this:

```xml
<Label 
                Text="Welcome to .NET Multi-platform App UI"
                Grid.Row="1"
                SemanticProperties.HeadingLevel="Level1"
                SemanticProperties.Description="Welcome to dot net Multi platform App U I"
                FontSize="18"
                HorizontalOptions="Center" />
```

When researching for accessibility options, I found [this Microsoft site](https://docs.microsoft.com/en-us/dotnet/maui/fundamentals/accessibility) that states you shouldn't use SemanticProperties.Description on Labels:

>Avoid setting the SemanticProperties.Description attached property on a Label. 
>This will stop the Text property being spoken by the screen reader. 
>This is because the visual text should ideally match the text read aloud by the screen reader.

Maybe someone should have a look at this. 
 

### Steps to Reproduce

1. Create a File > New .NET MAUI App
2. Examine MainPage.xaml

### Version with bug

Preview 14 (current)

### Last version that worked well

Preview 14 (current)

### Affected platforms

iOS, Android, Windows, macOS

### Affected platform versions

Independent of any version

### Did you find any workaround?

Just remove the SemanticProperties.Description attached property from labels if necessary.

**Comment by jfversluis** [comment]
Thanks for bringing this to our attention @Mephisztoe !

@rachelkang do you have any information on this?

**Comment by rachelkang** [comment]
Hi, @Mephisztoe - thanks for opening this issue and for reading into our docs! You are certainly correct in that we generally advise that `SemanticProperties.Description` be left unset on text-based controls including labels; this is because the text set by the `SemanticProperties.Description` overrides the visual text from being read aloud, creating a disparity between the visual and screen reader user experiences.

However, there are certain nuanced cases, as in this one, where setting a `SemanticProperties.Description` may actually be helpful rather than dissonant. In this case, it was unclear how various screen readers would process ".NET" and "UI" - would the "." be read as "dot", and "UI" as 2 separate letters? In order to avoid potential confusion, we provided a description for how the text should be read aloud by the screen reader, without changing what the text actually reads visually.

That being said, we are currently under the process of getting our templates reviewed for accessibility, so if the accessibility experts find that there is a better solution here, we'll find out and will be making any necessary changes soon. Hope that helps, and let me know if you have any questions!
